src/lib/proxyCache.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+const { createProxyCache } = require('@mojaloop/inter-scheme-proxy-cache-lib')
+const Config = require('./config.js')
+const ParticipantService = require('../../src/domain/participant')
+
+const proxyCache = createProxyCache(
+  Config.PROXY_CACHE_CONFIG.type,
+  Config.PROXY_CACHE_CONFIG.proxyConfig
+)
+const getCache = () => {
```

Comment:
What is the point of having this function?

Reply:
I know that **ioredis** lib have [lazyConnect](https://redis.github.io/ioredis/interfaces/CommonRedisOptions.html#lazyConnect) option, which is _false_ by default (when a Redis instance is initialized, a connection to the server is immediately established). 
In our lib I set it by default to _true_: https://mojaloop.github.io/inter-scheme-proxy-cache-lib/types/RedisProxyCacheConfig.html#Default%20Value

So it's a bit weird why proxyCache is still trying to connect as soon as you instantiate the connection object